packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -930,17 +939,27 @@ class _InkResponseState extends State<_InkResponseStateWidget>
     }
   }
 
+  bool get _hasSelectedOverlayColor {
+    const Set<MaterialState> selected = <MaterialState>{MaterialState.selected};
+    return widget.overlayColor?.resolve(selected) != null;
+  }
+
   void _updateFocusHighlights() {
     final bool showFocus;
+    final bool showSelect;
     switch (FocusManager.instance.highlightMode) {
       case FocusHighlightMode.touch:
         showFocus = false;
+        showSelect = true;
```

Comment:
For touch devices, do not display the focus color, try to display the selected color(setting by `overlayColor`) instead.

Reply:
Looks good

Reply:
Looks good